src/test/java/com/epam/definitions/ReflectionDefinitions.java

```diff
@@ -22,90 +17,49 @@
     private Object object;
 
     @Given("Class name - {string}")
```

Comment:
It might be more fluent if describe like
"Object is initialized using class - {string}"